apps/akka-kafka-streams-model-server/README.md

```diff
@@ -211,21 +235,21 @@ requires the following configuration parameters:
 
 ## Running Locally
 
-We can run the complete application from source, from the _docker_ images or with a combination of the two, depending of our goal.
+You can run the complete application from source, from the Docker images, or with a combination of the two, depending of your goals.
 
-For example, to test the end-to-end execution, we can opt to run all processes in _docker_ containers while if we are in the middle of a _develop-run-test-develop_ cycle of the _model serving_ part, we could run the `producer` in its container, while executing the server code from `sbt` or our IDE of choice.
+For example, to test the end-to-end execution, run all processes in Docker containers. If you are in the middle of a _develop-run-test_ cycle of the _model serving_ part, for example, you could run the `producer` Docker image, while executing the server code from `sbt` or your IDE of choice.
 
-In any case, the required external services should be reachable from the host executing the any of the components of the application.
+In any case, the required external services should be reachable from the host executing any of the components of the application.
 
-Note that when running against services installed on DC/OS, their DNS names are not resolvable and we should use IP addresses instead.
+Note that when running against services installed in a cluster with _virtual IP addressing_, the names may not be resolvable and IP addresses may be required instead.
 
-### Publisher: Running Local
+### Publisher: Running Locally
```

Comment:
I do not think this will work with Kubernetes. You can't access kubernetes network from outside. You can run it with the local Kafka broker, but not with the one in cluster

Reply:
Clarified wording that this is for running on a local machine only and you have to be able to access Kafka.